packages/portalnetwork/vitest.config.unit.ts

```diff
@@ -3,7 +3,18 @@ import { defineConfig } from 'vitest/config'
 export default defineConfig({
   test: {
     silent: true,
-    exclude: ['test/integration'],
+    exclude: [
+      'test/integration',
+      'test/networks/history/portalSpecTests.spec.ts',
+      'test/networks/state/content.spec.ts',
```

Comment:
Why these new exclusions?

Reply:
I think you just needed to run `git submodule -- init` or something.  They all are passing locally for me so I'm going to revert this change.